drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
Sorry, that was a copy-paste error which I just corrected. What I meant was something like

    client.get("http://www.example.com/").getEntrypoint(); 

or

    client.get("http://www.example.com/").getApiDocumentation().getEntrypoint();

The client would look for the Hydra HTTP header, fetch the API documentation, and return the entrypoint.